sites/skeleton.dev/src/routes/+layout.svelte

```diff
@@ -122,9 +123,9 @@
 	}
 
 	// Lifecycle
-	afterNavigate((params: any) => {
+	afterNavigate((params: AfterNavigate) => {
```

Comment:
```suggestion
	afterNavigate((params) => {
```
`params` can just be inferred so there's no need for importing `AfterNavigate`